starlette/config.py

```diff
@@ -78,7 +78,7 @@ def get(
 
     def _read_file(self, file_name: typing.Union[str, Path]) -> typing.Dict[str, str]:
         file_values: typing.Dict[str, str] = {}
-        with open(file_name) as input_file:
+        with open(file_name, encoding="utf8") as input_file:
```

Comment:
The filesystem encoding is supposed to be the same as the os.environ encoding. Should this actually be `errors="surrogateescape"`?